setup.py

```diff
@@ -16,7 +16,7 @@
     'six>=1.9.0',
     'waitress>=0.8.9',
     'cliquet[postgresql,monitoring]>=1.6.0',
-    'raven'
```

Comment:
I'm actually not sure why raven is present here (shouldn't it be directly pulled off from cliquet?)